examples/fungible-token/src/lib.rs

```diff
@@ -5,11 +5,15 @@
 *  - JSON calls should pass U128 as a base-10 string. E.g. "100".
 *  - The contract optimizes the inner trie structure by hashing account IDs. It will prevent some
 *    abuse of deep tries. Shouldn't be an issue, once NEAR clients implement full hashing of keys.
-*  - This contract doesn't optimize the amount of storage, since any account can create unlimited
-*    amount of allowances to other accounts. It's unclear how to address this issue unless, this
-*    contract limits the total number of different allowances possible at the same time.
-*    And even if it limits the total number, it's still possible to transfer small amounts to
-*    multiple accounts.
+*  - The contract tracks storage difference before and after the call. If the storage increases,
+*    the contract requires the caller of the contract to attach enough deposit to the function call
+*    to cover the storage stake difference.
```

Comment:
"the storage stake difference"?

Reply:
replaced to "the storage fee difference"